src/routes/jira/server/jira-server-url-post.ts

```diff
@@ -3,34 +3,71 @@ import axios from "axios";
 import { GitHubServerApp } from "models/github-server-app";
 import { isValidUrl } from "utils/is-valid-url";
 
+interface MessageAndCode {
+	errorCode: string;
+	message: string;
+	statusCode: number;
+}
+
+interface GheServerUrlErrors {
+	[key: string | number]: MessageAndCode;
+}
+
+export const gheServerUrlErrors: GheServerUrlErrors = {
+	invalidUrl: {
+		errorCode: "GHE_ERROR_INVALID_URL",
+		message: "Invalid URL",
+		statusCode: 200
+	},
+	ENOTFOUND: {
+		errorCode: "GHE_ERROR_ENOTFOUND",
+		message: "Request to URL failed",
+		statusCode: 200
+	},
+	502: {
+		errorCode: "GHE_SERVER_BAD_GATEWAY",
+		message: "Bad gateway",
+		statusCode: 502
```

Comment:
Even a bad gateway to github should probably return a 200 to the frontend.  Again, a 502 being returned from us means that _our api_ is having this issue, not github.  This will also mess with SLOs I think...

Reply:
I think all errors that can be caught should be 200s being returned